include/geode/mesh/core/regular_grid.h

```diff
@@ -122,6 +122,10 @@ namespace geode
 
         index_t vertex_index( const GridVertexIndex< dimension >& index ) const;
 
+        GridVertexIndex< dimension > cell_node_index(

```

Comment:
node vs vertex?

Reply:
nodes are local in each cell, vertices are global